pkg/config/model/viper.go

```diff
@@ -582,11 +586,30 @@ func (c *safeConfig) UnmarshalExact(rawVal interface{}) error {
 func (c *safeConfig) ReadInConfig() error {
 	c.Lock()
 	defer c.Unlock()
-	err := c.Viper.ReadInConfig()
+	err := errors.Join(c.Viper.ReadInConfig(), c.configSources[SourceFile].ReadInConfig())
 	if err != nil {
 		return err
 	}
-	return c.configSources[SourceFile].ReadInConfig()
+
+	// merge with extra config files
+	var errs []error
+	var successfulExtraConfigPaths []string
+	for _, path := range c.extraConfigFilePaths {
+		b, err := os.ReadFile(path)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("no config exists at %s", path))
```

Comment:
The error could be anything from read access error, file not existing  ...

```suggestion
			errs = append(errs, fmt.Errorf("could not read extra config file '%s': %s", path, err))
```

Reply:
Good point, done in 992729991b9716ac4c72dbd6e0b75dc679f3971c.